scripts/setup/globals.ps1

```diff
@@ -14,12 +14,16 @@ Param (
                  "DEFAULT_SERVICE_NAME",
                  "SERVICE_DESCRIPTION",
                  "CERT_EXPIRATION_WINDOW",
-                 "CERT_NAME")]
+                 "CERT_NAME",
+                 "IIS_ADMIN_API_OWNERS",
+                 "INSTALLER_FLAG",
+                 "IIS_ADMIN_API_OWNERS_DESCRIPTION")]
     [string]
     $Command
 )
 
 $INSTALL_METHOD_KEY = "IIS_ADMIN_INSTALL_METHOD"
+$installerFlag = "This item will be removed when Microsoft IIS Administration API is uninstalled."
```

Comment:
"This item will be removed **if the** Microsoft IIS Administration API is uninstalled."

What do you think about this?
"This group will be removed if the Microsoft IIS Administration API is uninstalled, but can be prevented if this sentence is removed."

Reply:
I have change the `when` to `if the`. Not sure if we want to suggest to the user to modify the description.